cesium_app/handlers/prediction.py

```diff
@@ -137,3 +139,28 @@ def delete(self, prediction_id):
         prediction = self._get_prediction(prediction_id)
         prediction.delete_instance()
         return self.success(action='cesium/FETCH_PREDICTIONS')
+
+
+class PredictSingleHandler(BaseHandler):
+    def post(self):
+        ts_data = tornado.escape.json_decode(self.get_argument('ts_data'))
+        model_id = int(self.get_argument('modelID'))
+        features_to_use = tornado.escape.json_decode(
```

Comment:
Aren't the features to use stored in the model? The user shouldn't be able to choose this.